python/lsst/ap/association/packageAlerts.py

```diff
@@ -36,10 +39,21 @@
 from lsst.pex.exceptions import InvalidParameterError
 import lsst.pipe.base as pipeBase
 from lsst.utils.timer import timeMethod
-
+import fastavro
+from lsst.alert.packet import Schema
+try:
+    from confluent_kafka import KafkaException
+    import confluent_kafka
+except ImportError as error:
+    error.msg += ("Could not import confluent_kafka. Alerts will not be sent to the alert stream")
 
 """Methods for packaging Apdb and Pipelines data into Avro alerts.
 """
+_ConfluentWireFormatHeader = struct.Struct(">bi")
+latest_schema = Schema.from_file().definition
```

Comment:
Don't load this here--you want to use the `schemaFile` set up in `PackageAlertsConfig`.

Reply:
Moved `_ConfluentWireFormatHeader` and swapped to `schemaFile`